### PR TITLE
LPS-171925, LPS-171927 | Test Automation | Master

### PR DESCRIPTION
**Pull request by davisaints**
**JIRA Tickest: 
[LPS-171925](https://issues.liferay.com/browse/LPS-171925)
[LPS-171927](https://issues.liferay.com/browse/LPS-171927)**

### CODE REVIEW

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 1]
```diff
@@ -1291,6 +1291,185 @@ definition {
 		}
 	}
 
+	@description = "Verify it's possible to update an object entry with the trigger On Order Status Update"
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:1294]: Can you add the ticket number here?

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 2]
```diff
@@ -1291,6 +1291,185 @@ definition {
...
+				newAddressCountry = "United States",
+				newAddressName = "Address Name",
+				newAddressZipCode = "Test Zip Code",
+				productName = "U-Joint",
+				productQuantity = 1,
+				productUnitPrice = 24,
+				shippingMethod = "Standard");
+		}
+
+		task ("The user changes payment status and the action changes order status to processing") {
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:1424]: Forgot the "When"

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 3]
```diff
@@ -1291,6 +1291,185 @@ definition {
...
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.assertObjectActions(
+				actionActive = "Yes",
+				actionLabel = "Custom Action",
+				status = "Success");
+		}
+	}
+
+	@description = "Verify it's possible to update an object entry with the trigger On Payment Status Update"
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:1375]: Ticket number needed

**Review by epark18 — COMMENTED** [review]
Great job on these. They just need the minor changes. Thank you!

**Comment by epark18** [comment]
ci:forward

**Pull request closed without merging**